test_util/test_runner.py

```diff
@@ -100,7 +100,7 @@ def integration_test(
         tunnel, test_dir,
         dcos_dns, master_list, agent_list, public_agent_list,
         variant, test_dns_search, ci_flags, timeout=None,
-        aws_access_key_id='', aws_secret_access_key='', region=''):
+        aws_access_key_id='', aws_secret_access_key='', region='', **add_env):
```

Comment:
add_env as given here will swallow anything / everything extra passed to this function. I'd much rather have us just pass through the add_env dictionary directly, rather than exploding it into proper parameters and going back again. The goal here is to pass through a very specific dictionary.